openfact.gemspec

```diff
@@ -46,6 +46,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'webmock', '~> 3.12'
   spec.add_development_dependency 'yard', '~> 0.9'
 
+  spec.add_runtime_dependency 'base64', '>= 0.1', '< 0.4'
   spec.add_runtime_dependency 'hocon', '~> 1.3'
+  spec.add_runtime_dependency 'logger', '~> 1.7'
```

Comment:
Very late and I know I previously approved, but I'm looking at packaging for Fedora now and I wonder why `~> 1.7`. https://stdgems.org/logger/ tells me that this means we can't use the version included in Ruby before. Is this really needed or can this be relaxed? It worked with the vendored version before so really, isn't this way too strict?

Do you recall why you chose this version?

Reply:
I chose it because it was the latest available one / the one that was used in tests. I don't mind changing it